pkg/config/config.go

```diff
@@ -361,7 +361,7 @@ func ProcessConfigForSpacelift() error {
 // https://medium.com/@bnprashanth256/reading-configuration-files-and-environment-variables-in-go-golang-c2607f912b63
 func processConfigFile(path string, v *viper.Viper) error {
 	if !u.FileExists(path) {
-		u.PrintInfoVerbose(fmt.Sprintf("No CLI config found in '%s'", path))
+		u.PrintInfoVerbose(fmt.Sprintf("No config file found at path '%s'.", path))
```

Comment:
Might as well mention atmos.yaml file name too

```suggestion
		u.PrintInfoVerbose(fmt.Sprintf("No config file atmos.yaml found in path '%s'.", path))
```